src/js/controllers/walletHome.js

```diff
@@ -46,18 +46,6 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
     self.resetForm();
   });
 
-  var disableOnlineListener = $rootScope.$on('Local/Online', function() {
-    // This is needed then the apps go to sleep
-    self.bindTouchDown();
-  });
-
-  var disableResumeListener = $rootScope.$on('Local/Resume', function() {
```

Comment:
"Tap event..." don't use events. This is called from view to the function: `index.openWallet(); index.updateTxHistory()`